### PR TITLE
PR for Authentication and Timer

### PR DESCRIPTION
**Pull request by manojasarsa**
This PR includes:
- Authentication
    - Login
    - SignUp
    - Logout
    - Forgot Password
- Timer for Quiz Questions
- Updated Readme.md file

### CODE REVIEW

File: auth/auth.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+}
+
+#login_button {
+    color: var(--white-color);
+}
+
+form .inp_checkbox {
+    column-gap: 0.6rem;
+}
+
```
Comment (BelideNikhil) [inline auth/auth.css]: use classname to target the form
as this might effect globally un-knowingly

**Review by BelideNikhil — APPROVED** [review]
Good to go!

File: auth/auth.css (hunk 1 above)
Comment (manojasarsa) [reply auth/auth.css]: sure, I'll change it :)

File: auth/auth.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    margin: 0 auto;
+    border: 1px solid var(--primary-color);
+}
+
+.logout_text {
+    margin: 1rem;
+}
+
+.logout_box .link {
+    color: var(--priamry-color);
```
Comment (MarkVed17) [inline auth/auth.css]: typo?

**Review by MarkVed17 — APPROVED** [review]
Rest all looks good! You may proceed.

File: auth/auth.css (hunk 2 above)
Comment (manojasarsa) [reply auth/auth.css]: oops yes! Thanks for pointing out, Vedant.

**Pull request merged**